linchpin/cli/__init__.py

```diff
@@ -382,7 +382,6 @@ def _execute_action(self, action, targets=(), run_id=None, tx_id=None):
 
         use_pinfile = True
         pf = None
-        pf_data = None
```

Comment:
this seems unrelated to this PR.  Do you need to rebase?

Reply:
This one actually affects a tests I was working on, it needs to be in here as it was reverted a second time by #577. This fixes that again.